cloudberry/zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/datastore/IPostTransform.scala

```diff
@@ -4,10 +4,12 @@ import play.api.libs.json.JsValue
 
 
 trait IPostTransform {
+  def category: String
   def transform(jsValue: JsValue): JsValue
 }
 
 case object NoTransform extends IPostTransform {
+  override def category: String = "default"
```

Comment:
If you need some class have the `category`, we should add a `trait` or `class` that inherits from the `IPostTransform` to be explicit.  E.g.
```
case class CategoricalWrap extends IPostTransform {
    def category: String
}
```

Reply:
Yes, we can achieve that in this way, but if so, the `handleRequest` of `BerryClient` parameter type has to be changed to `CategoricalWrap` as well. I am wondering whether it is good to change use cases of an interface instead of add a feature inside the interface.
What do you think? @JavierJia

Reply:
Interfaces should be succinct and self-explanable. We are refactoring towards a "correct" direction, not an "easier" direction. Most of the time, the "correct" will be "easier" ultimately.